cli/CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ _Released 7/30/2025 (PENDING)_
 - Fixed an issue where TypeScript ESM projects using `.js` and `.mjs` extensions where not resolving correctly within `@cypress/webpack-batteries-included-preprocessor`. Addressed in [#31994](https://github.com/cypress-io/cypress/pull/31994). Fixes [#26827](https://github.com/cypress-io/cypress/issues/26827) and [#28805](https://github.com/cypress-io/cypress/issues/28805).
 - Fixed an issue in `@cypress/angular` where component instance fields were not reference safe and were being overwritten. Fixes [#31238](https://github.com/cypress-io/cypress/issues/31238) and [#31983](https://github.com/cypress-io/cypress/issues/31983). Fixed in [#31993](https://github.com/cypress-io/cypress/pull/31993).
 
+**Dependency Updates:**
+
+- Pinned `form-data` to `4.0.4`. This removes the [SNYK-JS-FORMDATA-10841150](https://security.snyk.io/vuln/SNYK-JS-FORMDATA-10841150) vulnerability being reported in security scans. Addresses [#32066](https://github.com/cypress-io/cypress/issues/32066).
```

Comment:
This is no longer correct, and there is no dependency update, only a lock file update at this time. Lockfiles are not published to the npm registry.

The only change at the moment will influence how the Cypress binary is built. It doesn't change anything in the published package.json.

If that is all you want to merge, then I suggest to merge PR https://github.com/cypress-io/cypress/pull/32083 instead of this one.